### PR TITLE
fixed most problems with check_message_status tests

### PR DESCRIPTION
**Pull request by jstavel**
the changes do:
- throw a SkipException when no subscription is available
- new test that verifies that a dialog with an error is risen when attaching temporary subscription with future date
-

### CODE REVIEW

**Comment by Lorquas** [comment]
thumbs up from me

File: src/rhsm/gui/tests/subscription_status_tests.clj [hunk 1]
```diff
@@ -94,64 +95,106 @@
   check_status_message_after_attaching
   "Asserts that status message displayed in main-window is right after attaching subscriptions"
   [_]
-  (try
+  (tasks/search :match-installed? true)
+  (when (not (bool (tasks/ui guiexist :all-subscriptions-view)))
```
Comment (rarebreed) [inline src/rhsm/gui/tests/subscription_status_tests.clj]: Is this a proper skip?  I'm wondering if there is another possible way for the :all-subscriptions-view to be empty other than if there are only temporary subscriptions?

**Comment by jstavel** [comment]
I think that it does not make sense to use those tests when no subscription is available.
Because those tests plays with attaching and they verify that sub-man offers actual informations (num of products not subscribed, ...)

File: src/rhsm/gui/tests/subscription_status_tests.clj (hunk 1 above)
Comment (Lorquas) [inline src/rhsm/gui/tests/subscription_status_tests.clj]: yeah maybe you could do a count of the subscriptions view to see if its empty

**Pull request merged**